site/content/en/docs/drivers/qemu.md

```diff
@@ -29,10 +29,17 @@ minikube start supports some qemu specific flags:
   * Note: while the flag should override the config, if the flag does not take effect try running `minikube delete`.
   * MacPorts: if you are installing [minikube](https://ports.macports.org/port/minikube/) and [qemu](https://ports.macports.org/port/qemu/) via MacPorts on a Mac with M1, use the following flag: `--qemu-firmware-path=/opt/local/share/qemu/edk2-aarch64-code.fd`
 
-## Issues
+## Known Issues
 
 * [Full list of open 'qemu' driver issues](https://github.com/kubernetes/minikube/labels/co%2Fqemu-driver)
 
+* When using the `user` network (default) the guest uses **only** the first `nameserver` entry in the hosts `/etc/resolv.conf` for DNS lookup.
```

Comment:
Thank you for this quick PR when looking the issue on the live page, it was hard for me to figure out if this is one issue or multiple and what is the work arround,

and also I think it is better to have a Bigger Font or numbered header for the issue
something like this for the header
## "1 - Qemu start stuck , on Corp machines or custom DNS...."
Description of the issue in a paragraph, Description of the issue in a paragraphDescription of the issue in a paragraphDescription of the issue in a paragraphDescription of the issue in a paragraphDescription of the issue in a paragraph.

#### Workarounds/Solution:
1- Try blahblah dedicated driver...
2- Try ...


## 2. Second Issue (if any

Reply:
Sounds good, updated the format